aiven_db_migrate/migrate/pgutils.py

```diff
@@ -105,3 +109,36 @@ def parse_connection_string_url(url: str) -> Dict[str, str]:
     for k, v in parse_qs(p.query).items():
         fields[k] = v[-1]
     return fields
+
+
+def wait_select(conn, timeout=None):
+    start_time = time.monotonic()
```

Comment:
I would appreciate a comment why this is needed (docstring or above where it is used) -> Before it linked https://www.psycopg.org/docs/faq.html#faq-interrupt-query